service/frontend/admin/handler.go

```diff
@@ -1293,9 +1293,13 @@ func (adh *adminHandlerImpl) ResendReplicationTasks(
 	if request == nil {
 		return adh.error(validate.ErrRequestNotSet, scope)
 	}
+	remoteAdminClient, err := adh.GetRemoteAdminClient(request.GetRemoteCluster())
+	if err != nil {
+		return adh.error(err, scope)
```

Comment:
nit: this should be classified as bad request potentially to avoid impacting SLA